packages/fxa-payments-server/package.json

```diff
@@ -76,6 +76,7 @@
     "prettier": "^1.18.2",
     "redux-devtools-extension": "^2.13.8",
     "sass-lint": "^1.13.1",
+    "supertest": "4.0.2",
```

Comment:
Actually: Never mind me - this *does* work within jest. For some reason, I thought this was a new test framework and not a tool that works within the framework. Probably a fine choice by virtue of 1) it works and 2) we didn't have any server route tests like this already.

Reply:
Yeah, still just. Just adding supertest for the route calls.